csi/deployment.go

```diff
@@ -35,6 +36,9 @@ const (
 
 	AnnotationCSIGitCommit = types.LonghornDriverName + "/git-commit"
 	AnnotationCSIVersion   = types.LonghornDriverName + "/version"
+
+	SoftCSIPodAntiAffinityPreset = "soft"
+	HardCSIPodAntiAffinityPreset = "hard"
```

Comment:
Suggst to using a common prefix so related constants are easier to find and recognize.